tests/Command/CopyConnectionSchemaCommandTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+declare(strict_types=1);
+
+namespace Kununu\TestingBundle\Tests\Command;
+
+use Doctrine\DBAL\Connection;
+use Kununu\TestingBundle\Command\CopyConnectionSchemaCommand;
+use Kununu\TestingBundle\Service\SchemaCopy\SchemaCopyAdapterFactoryInterface;
+
+/**
+ * @group legacy
```

Comment:
Why are all these command tests marked as legacy?

Reply:
To avoid deprecation warnings that would have the tests to fail